provisioning/service/samples/readme.md

```diff
@@ -1,14 +1,15 @@
-Service provisioning samples were moved to [Azure-Samples/azure-iot-samples-csharp][samples-repo] repository
+## Provisioning Device Sample - Microsoft Azure IoT SDK for .NET
 
-### [Service provisioning samples][service-prov-samples]
-* [GroupCertificateVerificationSample][group-cert-sample]
-* [BulkOperationSample][bulk-op-sample]
-* [EnrollmentSample][enrollment-sample]
-* [EnrollmentGroupSample][enrollment-group-sample]
+### Service provisioning samples
 
-[samples-repo]: https://github.com/Azure-Samples/azure-iot-samples-csharp
-[service-prov-samples]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/main/provisioning/Samples/service
-[group-cert-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/main/provisioning/Samples/service/GroupCertificateVerificationSample
-[bulk-op-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/main/provisioning/Samples/service/BulkOperationSample
-[enrollment-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/main/provisioning/Samples/service/EnrollmentSample
-[enrollment-group-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/main/provisioning/Samples/service/EnrollmentGroupSample
\ No newline at end of file
+* [Group certificate verification sample][group-cert-sample]
+* [Bulk operation sample][bulk-op-sample]
+* [Enrollment sample][enrollment-sample]
+* [Enrollment group sample][enrollment-group-sample]
+* [Clean up enrollments sample][clean-up-enrollments-sample]
```

Comment:
Might as well standardize on one kind of markdown bullet.

Also, I find that if all the text is a link it can get visually overwhelming. Can we link only part of it?

```suggestion
- [Group certificate verification][group-cert-sample] sample
- [Bulk operation][bulk-op-sample] sample
- [Enrollment][enrollment-sample] sample
- [Enrollment group][enrollment-group-sample] sample
- [Clean up enrollments][clean-up-enrollments-sample] sample
```